lib/runCall.js

```diff
@@ -18,6 +18,7 @@ const ERROR = exceptions.ERROR
  * @param opts.gasPrice {Buffer}
  * @param opts.origin {Buffer} []
  * @param opts.to {Buffer}
+ * @param opts.txHash {Buffer}
```

Comment:
Hmm, not sure, maybe I don't understand this fully, but why does ``txHash`` also goes here (and in ``runCode`` as well) go in the options?

Reply:
sorry for late reply and thanks for reviewing!
`runCode.js` emits `step` event here
https://github.com/ethereumjs/ethereumjs-vm/blob/master/lib/runCode.js#L128
And runBlock calls runTx -> runCall -> runCode, but `runCode` and `runCall` don't know txHash, so we need pass txHash to runCall.